superset-frontend/src/components/FaveStar.tsx

```diff
@@ -59,7 +60,7 @@ export default class FaveStar extends React.PureComponent<FaveStarProps> {
                   ? 'favorite-selected'
                   : 'favorite-unselected'
               }
-              viewBox="0 0 16 15"
+              viewBox={this.props.viewBox || '0 0 16 16'}
```

Comment:
ah good catch, I didn't notice that. All the other icons have a size of 24 and a viewbox of `0 0 24 24`, so we should probably standardize around that and override where necessary.

Reply:
Good catch, I'm not sure what I was thinking. Looks like it's not actually applying this for my use case since I didn't add a tooltip prop. Reverting this back...